Mathlib/Analysis/InnerProductSpace/PiL2.lean

```diff
@@ -1067,18 +1113,33 @@ lemma toEuclideanLin_eq_toLin_orthonormal [Fintype m] :
 end Matrix
 
 local notation "⟪" x ", " y "⟫ₑ" =>
+  @inner 𝕜 _ _ (WithLp.toLp 2 x) (WithLp.toLp 2 y)
+
+local notation "⟪" x ", " y "⟫ₑ'" =>
   @inner 𝕜 _ _ (Equiv.symm (WithLp.equiv 2 _) x) (Equiv.symm (WithLp.equiv 2 _) y)
 
 /-- The inner product of a row of `A` and a row of `B` is an entry of `B * Aᴴ`. -/
 theorem inner_matrix_row_row [Fintype n] (A B : Matrix m n 𝕜) (i j : m) :
     ⟪A i, B j⟫ₑ = (B * Aᴴ) j i := by
   rfl
 
+/-- The inner product of a row of `A` and a row of `B` is an entry of `B * Aᴴ`. -/
+@[deprecated inner_matrix_row_row (since := "2024-04-27")]
+theorem inner_matrix_row_row' [Fintype n] (A B : Matrix m n 𝕜) (i j : m) :
+    ⟪A i, B j⟫ₑ' = (B * Aᴴ) j i := by
+  rfl
```

Comment:
I don't see the point of adding a new lemma that's immediately deprecated. You are changing the lemma statement of `inner_matrix_row_row`, so whatever downstream uses of `inner_matrix_row_row` there are, creating a new lemma `inner_matrix_row_row'` won't save the people from having to figure out what happened.
```suggestion
```
Same below